roofit/xroofit/inc/RooBrowser.h

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Also this file needs a header guard. Maybe you could follow the RooFit convention here:
```C++
#ifndef RooFit_RooBrowser_h
#define RooFit_RooBrowser_h
```
also the `endif` at the end of the file.